src/lib/pages/frontsite/Home/Sections/SectionInstallation.js

```diff
@@ -0,0 +1,139 @@
+import React, { Component } from 'react';
+import {
+  Card,
+  Container,
+  Header,
+  List,
+  Button,
+  Grid,
+  Message,
+  Icon,
+  Label,
+} from 'semantic-ui-react';
+
+export default class SectionInstallation extends Component {
+  render() {
+    return (
+      <Container fluid className="dot-background-container">
+        <Container fluid className="dot-background">
+          <Container className="fs-landing-page-section">
+            <Header
+              as="h1"
+              className="section-header highlight"
+              textAlign="center"
+            >
+              Installation
+            </Header>
+            <Card.Group itemsPerRow={3} stackable className="install-cards">
+              <Card color="orange" className="install-card">
+                <Card.Content>
+                  <Header size="medium">
+                    <Label circular size="huge" color="orange">
+                      1
+                    </Label>{' '}
+                    Install Backend
+                  </Header>
+                  <Card.Description>
+                    <List>
+                      <List.Item className="install-item">
```

Comment:
Shouldn't all the `List.Item` have the same style? `install-item`, spacing elements is nice!